pgbouncer/tox.ini

```diff
@@ -0,0 +1,39 @@
+[tox]
+minversion = 2.0
+basepython = py27
+envlist = pgbouncer-{1_7,1_8}, flake8
+
+[common]
```

Comment:
Let's get rid of this section and put `commands` under `[testenv]` for default inheritance.